src/views/Generator/ValidatorDialog.tsx

```diff
@@ -79,6 +91,14 @@ export function ValidatorDialog({
               <Flex align="center">
                 Validator
                 {isRunning && <Spinner ml="2" />}
+                {!isProxyHealthy && (
+                  <Tooltip content="Grafana k6 Studio cannot establish connection to the Internet. Unless this is expected due to your internal network configuration, check your proxy settings.">
+                    <Badge color="orange" ml="2">
+                      <TriangleAlert />
+                      Proxy health check failed
+                    </Badge>
+                  </Tooltip>
```

Comment:
We don't have an idle state in this component, so we opted for using a Badge component instead of the Callout. Any thoughts?

<img width="507" alt="image" src="https://github.com/user-attachments/assets/1e2707e8-889e-4136-b7d9-f6739f9009b2" />